src/clj/rems/auth/oidc.clj

```diff
@@ -76,10 +78,14 @@
         extra-attributes (select-keys id-data (map (comp keyword :attribute) (:oidc-extra-attributes env)))
 
         user-info (when-let [url (:userinfo_endpoint oidc-configuration)]
-                    (-> (http/get url
-                                  {:headers {"Authorization" (str "Bearer " access-token)}})
-                        :body
-                        json/parse-string))
+                    (try
+                      (-> (http/get url
+                                    {:headers {"Authorization" (str "Bearer " access-token)}})
+                          :body
+                          json/parse-string)
+                      (catch Exception e
+                        (log/warn "Fetching user info failed" e)
+                        {})))
```

Comment:
Should this be fatal or not?

Reply:
This change makes it non-fatal, which was needed since ADFS was refusing to give us userinfo. At the moment userinfo is only needed for bona fide status, so it's fine if it's missing. Better to let the user in, I think...

Reply:
Well if we let a handler in, we will not know their elixir id / researcher status or whatever that we were supposed to get and will get confusing errors?

Reply:
I guess one alternative is to make this configurable via `:oidc-fetch-user-info` or something. Another alternative is to make REMS work with ADFS and leave this as a fatal error.

Reply:
So maybe we will make the fetching configurable and by default off so we can have a stable `.2` release and configure bona fide REMS correctly. We can come up with a more elaborate strategy later?